terraform/README.md

```diff
@@ -0,0 +1,35 @@
+# Mondoo Terraform Action
```

Comment:
We should sprinkle HashiCorp in here for search

Reply:
Do you mean `# Mondoo HashiCorp Terraform Action` or in the text below? I would vote for option 2 but let me know your thoughts.

Reply:
It can be anywhere. Just make sure we have the word somewhere in this.

Reply:
👍